packages/types/lib/capabilities.ts

```diff
@@ -15,16 +16,6 @@ export type BaseCapabilities = ConstraintsToCaps<BaseDriverCapConstraints>;
  */
 export type BaseNSCapabilities = CapsToNSCaps<ConstraintsToCaps<BaseDriverCapConstraints>>;
 
-/**
- * These may (or should) be reused by drivers.
- */
-export type AppiumAndroidCapabilities = WdioCaps.AppiumAndroidCapabilities;
```

Comment:
are these not used anywhere else?

Reply:
not that I'm aware of, no.  the intent was originally for a driver to consume it, but since capabilities can now be derived via constraints, there's no real point to keeping it.  I certainly didn't send a PR into e.g., uiautomator2 to use this